cypress/e2e/swap.test.ts

```diff
@@ -13,7 +13,7 @@ describe('Swap', () => {
   it('can enter an amount into input', () => {
     cy.get('#swap-currency-input .token-amount-input')
       .clear()
-      .type('0.001', { delay: 200 })
+      .type('0.001', { delay: 1000 })
```

Comment:
we probably should figure out why the page reloads but i'm just deciding to move on. i'll add a todo tho